package.json

```diff
@@ -27,35 +27,31 @@
     "@octokit/core": "^4.0.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.21.4",
+    "@babel/plugin-transform-modules-commonjs": "^7.21.2",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-typescript": "^7.21.4",
+    "@jest/globals": "^29.5.0",
     "@octokit/core": "^4.0.0",
     "@octokit/request-error": "^3.0.0",
     "@pika/pack": "^0.3.7",
     "@pika/plugin-build-node": "^0.9.1",
     "@pika/plugin-build-web": "^0.9.1",
     "@pika/plugin-ts-standard-pkg": "^0.9.1",
     "@types/fetch-mock": "^7.3.1",
-    "@types/jest": "^29.0.0",
+    "@types/jest": "^29.5.1",
     "@types/node": "^18.0.0",
+    "babel-jest": "^29.5.0",
     "fetch-mock": "^9.0.0",
     "github-openapi-graphql-query": "^4.0.0",
-    "jest": "^29.0.0",
+    "jest": "^29.5.0",
     "npm-run-all": "^4.1.5",
     "prettier": "2.8.7",
     "semantic-release-plugin-update-version-in-files": "^1.0.0",
-    "ts-jest": "^29.0.0",
+    "ts-jest": "^29.1.0",
+    "ts-node": "^10.9.1",
     "typescript": "^5.0.0"
   },
-  "jest": {
```

Comment:
When running the `jest` command it was throwing an error and running the tests in the `src` folder. Completely ignoring the `tests` folder after updating the jest version.  `jest.config.ts` replaced this.